bears/general/FilenameBear.py

```diff
@@ -71,15 +81,32 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if new_name != filename_without_extension:
+            messages.append(
+                'Filename does not follow {} naming-convention.'.format(
+                    file_naming_convention))
+
+        if not filename_without_extension.startswith(filename_prefix):
+            new_name = filename_prefix + new_name
+            messages.append(
+                'Filename does not use the {!r} prefix.'.format(
+                    filename_prefix))
+
+        if not filename_without_extension.endswith(filename_suffix):
+            new_name = new_name + filename_suffix
+            messages.append(
+                'Filename does not use the {!r} suffix.'.format(
+                        filename_suffix))
```

Comment:
Well, the error is reproducible on my local env.
Anyway, i will unindent lines 93 and 99 as requested and push the changes without running coala.
Let's see what will gitmate say :smile: